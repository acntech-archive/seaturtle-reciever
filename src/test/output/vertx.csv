Percentage served,Time in ms
0,0.841
1,1.554
2,1.566
3,1.573
4,1.576
5,1.578
6,1.581
7,1.585
8,1.588
9,1.589
10,1.593
11,1.595
12,1.597
13,1.598
14,1.600
15,1.601
16,1.604
17,1.606
18,1.608
19,1.610
20,1.614
21,1.616
22,1.618
23,1.622
24,1.624
25,1.626
26,1.628
27,1.630
28,1.632
29,1.636
30,1.639
31,1.642
32,1.644
33,1.647
34,1.651
35,1.654
36,1.656
37,1.660
38,1.662
39,1.665
40,1.667
41,1.670
42,1.672
43,1.677
44,1.680
45,1.682
46,1.685
47,1.690
48,1.694
49,1.698
50,1.701
51,1.706
52,1.712
53,1.719
54,1.728
55,1.754
56,1.776
57,1.793
58,1.832
59,1.863
60,1.960
61,1.995
62,2.024
63,2.050
64,2.085
65,2.146
66,2.262
67,2.395
68,2.498
69,2.549
70,2.583
71,2.652
72,2.722
73,2.774
74,2.835
75,2.974
76,3.052
77,3.273
78,3.578
79,3.710
80,3.763
81,3.863
82,3.963
83,4.051
84,4.221
85,4.292
86,4.390
87,4.456
88,4.520
89,4.586
90,4.715
91,4.882
92,5.022
93,5.292
94,5.543
95,5.770
96,6.645
97,8.464
98,10.192
99,14.679
100,25.085
