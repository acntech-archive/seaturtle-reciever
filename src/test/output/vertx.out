This is ApacheBench, Version 2.3 <$Revision: 1706008 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking localhost (be patient)
Completed 100 requests
Completed 200 requests
Completed 300 requests
Completed 400 requests
Completed 500 requests
Completed 600 requests
Completed 700 requests
Completed 800 requests
Completed 900 requests
Completed 1000 requests
Finished 1000 requests


Server Software:        
Server Hostname:        localhost
Server Port:            8080

Document Path:          /seaturtle/api/heartbeat
Document Length:        3954 bytes

Concurrency Level:      10
Time taken for tests:   0.276 seconds
Complete requests:      1000
Failed requests:        0
Keep-Alive requests:    1000
Total transferred:      4066000 bytes
HTML transferred:       3954000 bytes
Requests per second:    3629.07 [#/sec] (mean)
Time per request:       2.756 [ms] (mean)
Time per request:       0.276 [ms] (mean, across all concurrent requests)
Transfer rate:          14409.94 [Kbytes/sec] received

Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        0    0   0.1      0       1
Processing:     1    3   2.4      2      25
Waiting:        1    3   2.4      2      25
Total:          1    3   2.4      2      25

Percentage of the requests served within a certain time (ms)
  50%      2
  66%      2
  75%      3
  80%      4
  90%      5
  95%      6
  98%     10
  99%     15
 100%     25 (longest request)
